index.js

```diff
@@ -58,17 +60,20 @@ function HomebridgeRoutineTriggerAccessory(log, config) {
               if(body && body.message) { errorMsg = body.message; }
               if(body && body.msg) { errorMsg = body.msg; }
 
-              accessory.log("Error '" + response.statusCode + "': " + errorMsg);
+              console.log("Error '" + response.statusCode + "': " + errorMsg);
```

Comment:
`console.log` shouldn't be used, should be a contextual log reference `this.log`